### PR TITLE
web_unsplash traceback on saving the product

### PR DESCRIPTION
**Pull request by msh-odoo**
Task:https://www.odoo.com/web#id=2063293&action=333&active_id=1251&model=project.task&view_type=form&menu_id=4720

Pad: https://pad.odoo.com/p/r.c134d9b0d8ea08eb773491d23bb96507

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web_unsplash/models/ir_qweb.py [hunk 1]
```diff
@@ -8,6 +8,8 @@ class Image(models.AbstractModel):
 
     @api.model
     def from_html(self, model, field, element):
+        if element.find('.//img') is None:
```
Comment (rdeodoo) [inline addons/web_unsplash/models/ir_qweb.py]: Why is the condition not checking the same xpath as the one used after?
IMO
```python
if element.find('.//img') is None:
```
Should target same element:
```python
if element.find('img') is None:
```

Comment (msh-odoo) [reply addons/web_unsplash/models/ir_qweb.py]: It is due to copy paste from master branch I created previously ;)

**Comment by rdeodoo** [comment]
@JKE-be Can you r+ this one?

**Comment by JKE-be** [comment]
@robodoo r+

**Pull request closed without merging**